decidim-admin/app/packs/entrypoints/decidim_admin_overrides.scss

```
@@ -0,0 +1,2 @@
+// Application specific styles: https://docs.decidim.org/en/customize/styles/#_webpacker
```

Let's add a link to the whole page as this is currently explained in the other section (`Overriding CSS and styles`)
```suggestion
// Application specific styles: https://docs.decidim.org/en/customize/styles/
```